api/src/pcapi/core/offers/api.py

```diff
@@ -193,10 +193,26 @@ def _get_internal_accessibility_compliance(venue: offerers_models.Venue) -> dict
     }
 
 
+def _get_coherent_venue_with_subcategory(
+    venue: offerers_models.Venue, offer_subcategory_id: str
+) -> offerers_models.Venue:
+    # FIXME: ogeber 30.08.2024 - This wont be useful when
+    # virtual venues will be removed
+    subcategory = subcategories.ALL_SUBCATEGORIES_DICT[offer_subcategory_id]
+    if not subcategory.is_online_only and not venue.isVirtual:
```

Comment:
Question : peut-on se retrouver dans le cas `not subcategory.is_online_only and venue.isVirtual` ?
Selon ma compréhension, on se retrouverait alors à faire une requête pour trouver un lieu virtuel alors qu'il l'est déjà, pour une offre qui n'est pas « _online only_ ».  Est-ce le comportement souhaité ?

Reply:
Le front ne renvoie plus que les venues physiques (ie. `venue.isVirtual = False`) lors de la création d'offre. Si il n'y a pas de venue physique associé à l'offre, le front renvoie également une erreur à l'utilisateur lui indiquant qu'il lui faut créer un lieu physique pour publier une offre physique; 

Mais je peux faire un petit `raise` au cas où on a une sous catégorie non virtuelle et que le front renvoie un lieu virtuel (ce qui ne devrait jamais arriver)